python/tests/transformers/named_image_test.py

```diff
@@ -74,19 +78,30 @@ def setUpClass(cls):
         imageArray = np.empty((len(images), shape[0], shape[1], 3), 'uint8')
```

Comment:
let's move `getSampleImagelist()` as a "private" method in this test class and have it take care of resizing & reverseChannels as well. so a reader doesn't have to worry about `getSampleImageList`'s return image type.